libavcodec/x86/vvc/vvc_deblock.asm

```diff
@@ -41,7 +41,7 @@ cextern pw_4096
 cextern pw_8192
 cextern pw_m1
 cextern pd_1
-cextern pd_3
+pd_3 :           times 4 dd  3
```

Comment:
after the refact pd_3 and some others are not needed. 
see https://github.com/ffvvc/FFmpeg/pull/278/files#diff-519e64dabcabdb9e30163dbbd4c91c2037fea9304dcbc2da1313fc6044c4f1c4R42